### ISSUE TITLE
System to prevent kill stealing

### ISSUE THREAD

**Issue by FieryAU** [WarzoneMC/Warzone#113]
Basically, I've been getting a lot of my kills stolen by my teammates and I'm trying to climb the leaderboards, kill stealing is a common nuisance.

My idea to fix this is by creating a system where the player who deals the most player to the enemy gets the kill, but if other people also fought them but didn't deal much damage, they would get a point for an assist.

This could also bring a new section to the leaderboards for kill assists.

**Comment by Whnd** [comment]
agree

**Comment by Synconix** [comment]
Agreed, maybe like "You assisted (player) on killing (opposite player)"

**Comment by Virtualizes** [comment]
Yeah
